README.md

```diff
@@ -509,25 +508,25 @@ bin/foreman-installer --foreman-db-password=UNDEF
 
 It also works in interactive mode.
 
-You may also need to override array parameter with empty array value. For this
+You may also need to override array parameters with empty array values. For this
 purpose you can use `EMPTY_ARRAY` string as a value. Similarly you can use
 `EMPTY_HASH` for hash parameters.
 
 ## Hooks
 
-You may need to add new features to the installer. Kafo provides simple hook
-mechanism that allows you to run custom code at various time. We support
+You may need to add new features to the installer. Kafo provides a simple hook
+mechanism that allows you to run custom code at any time. We support
```

Comment:
`any` is maybe too strong word, we have 6 certain points in kafo run where we can hook custom code